internal/kafka/internal/workers/clusters_mgr.go

```diff
@@ -1245,3 +1248,22 @@ func (c *ClusterManager) setKafkaPerClusterCountMetrics() error {
 
 	return nil
 }
+
+func (c *ClusterManager) setClusterProviderResourceQuotaMetrics() error {
+	ocmCredentialsSpecified := c.OCMConfig.CheckIfCredentialsAreProvided()
+	if ocmCredentialsSpecified {
```

Comment:
we could do an early return here by inversing the condition.
```suggestion
	if !ocmCredentialsSpecified {
```

what do you think? @JameelB @JasonPowr

Reply:
Yeah that makes sense. 

Just note that this will need to be modified if we ever add new providers that we want to expose this metric for since we wouldn't want to return straight away in that case. This is probably far in the future though :)